openupgrade_scripts/scripts/stock/15.0.1.1/post-migration.py

```diff
@@ -0,0 +1,128 @@
+from openupgradelib import openupgrade
+
+
+def _fill_product_template_detailed_type(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE product_template
+        SET detailed_type = type
+        WHERE type = 'product'
+        """,
+    )
+
+
+def _fill_stock_move_is_inventory(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE stock_move
+            SET is_inventory = True
+            Where inventory_id IS NOT NULL;
+        """,
+    )
+
+
+def _fill_stock_picking_type_print_label(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE stock_picking_type
+            SET print_label = True
+            WHERE code = 'outgoing';
+        """,
+    )
+
+
+def _create_default_return_type_for_all_warehouses(env):
+    all_warehouses = env["stock.warehouse"].with_context(active_test=True).search([])
```

Comment:
something went wrong
```suggestion
    all_warehouses = env["stock.warehouse"].with_context(active_test=False).search([])
```